src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs

```diff
@@ -0,0 +1,53 @@
+using HotChocolate.Execution.Processing;
+
+namespace HotChocolate.Data.Projections.Handlers
+{
+    /// <summary>
+    /// This wrapper is used to combined two interceptors and create a chain
+    /// </summary>
+    internal class ProjectionInterceptorCombinator<T>
+        : IProjectionFieldInterceptor<T>
+        where T : IProjectionVisitorContext
+    {
+        private readonly IProjectionFieldInterceptor _first;
+        private readonly IProjectionFieldInterceptor _second;
+
+        public ProjectionInterceptorCombinator(
+            IProjectionFieldInterceptor first,
+            IProjectionFieldInterceptor second)
+        {
+            _first = first;
+            _second = second;
+        }
+
+        public bool CanHandle(ISelection selection) => true;
+
+        public void BeforeProjection(
+            T context,
+            ISelection selection)
+        {
+            if (_first is IProjectionFieldInterceptor<T> firstHandler)
+            {
+                firstHandler.BeforeProjection(context, selection);
+            }
+
+            if (_second is IProjectionFieldInterceptor<T> secondHandler)
+            {
+                secondHandler.BeforeProjection(context, selection);
+            }
+        }
+
+        public void AfterProjection(T context, ISelection selection)
+        {
+            if (_first is IProjectionFieldInterceptor<T> firstHandler)
+            {
+                firstHandler.AfterProjection(context, selection);
+            }
+
+            if (_second is IProjectionFieldInterceptor<T> secondHandler)
+            {
+                secondHandler.AfterProjection(context, selection);
+            }
```

Comment:
Should I flip these @michaelstaib ?

Reply:
Like 
BeforeProjection: first, second
AfterProjection: second, first

Reply:
I will also rename second to next in this case

Reply:
yes, seems more logical ....

Reply:
??

Like
BeforeProjection: first, second
AfterProjection: second, first

Reply:
Didn't you want to flip?

Reply:
i do, thought i did commit..
maybe did not push